app/templates/components/forms/admin/settings/system/mail-settings.hbs

```diff
@@ -0,0 +1,89 @@
+<h3 class="ui header">
+  {{t 'Mail Settings'}}
+</h3>
+<div class="field">
+  <label>
+    {{t 'From Email'}}
+  </label>
+  {{input type='text' name='email_from'}}
+</div>
+<div class="field">
+  <label>
+    {{t 'From Name'}}
+  </label>
+  {{input type='text' name='email_from_name'}}
+</div>
+<div class="ui hidden divider"></div>
+<div class="field">
+  {{ui-radio label=(t 'Disable email') name='mail_setting' value='disable' current='disable' onChange=(action (mut selectedSetting))}}
+</div>
+<div class="field">
+  {{ui-radio label=(t 'SMTP') name='mail_setting' value='smtp' onChange=(action (mut selectedSetting))}}
+</div>
+<div class="{{if (not-eq selectedSetting 'smtp') 'system hidden'}}">
+  <div class="field">
+    <label>
+      {{t 'Host'}}
+    </label>
+    {{input type='text' name='smtp_host'}}
+  </div>
+  <div class="field">
+    <label>
+      {{t 'Port'}}
+    </label>
+    {{input type='text' name='smtp_port'}}
+  </div>
+  <div class="less padding field">
+    <label>{{t 'Encryption'}}</label>
+    {{#ui-dropdown class='fluid selection' }}
+      {{input type='hidden' id='encryption_select'}}
+      <div class="default text">
+        {{t 'Choose Encryption type'}}
+      </div>
+      <i class="dropdown icon"></i>
+      <div class="menu">
+        <div class="item" data-value="tls">
+          {{t 'TLS'}}
+        </div>
+        <div class="item" data-value="tls-optional">
+          {{t 'TLS (optional)'}}
+        </div>
+        <div class="item" data-value="ssl">
+          {{t 'SSL'}}
+        </div>
+        <div class="item" data-value="no-encryption">
+          {{t 'No Encryption'}}
+        </div>
+      </div>
+    {{/ui-dropdown}}
+  </div>
+  <div class="field">
+    <label>
+      {{t 'Username'}}
+    </label>
+    {{input type='text' name='smtp_username'}}
+  </div>
+  <div class="field">
+    <label>
+      {{t 'Password'}}
+    </label>
+    {{input type='password' name='smtp_password'}}
+  </div>
+  <div class="ui hidden divider"></div>
+</div>
+<div class="field">
+  {{ui-radio label=(t 'Sendgrid API') name='mail_setting' value='sendgrid' onChange=(action (mut selectedSetting))}}
+  <span class="small text">
+    ({{t 'See'}}
+    <a href="https://sendgrid.com/docs/User_Guide/Settings/api_keys.html" target="_blank" rel="noopener nofollow">
+      {{t 'here'}}
+    </a>
+    {{t 'on how to get this token.'}})
+  </span>
+</div>
+<div class="{{if (not-eq selectedSetting 'sendgrid') 'system hidden'}} field">
```

Comment:
Don't use a hidden class. Use a full if statement and don't render the things not required.